pkg/porter/testdata/cache/README.md

```diff
@@ -1,5 +1,5 @@
 # Cache Cheat Sheet
 
 * deislabs/kubekahn:1.0 -> 887e7e65e39277f8744bd00278760b06
-* getporter/mysql:v0.1.2 -> 972978d3b715212b783a70fdc3449584
+* getporter/mysql:v0.1.3 -> 972978d3b715212b783a70fdc3449584
```

Comment:
Oops, this should not have been changed. The cache keys in porter correspond to the bundle tag, so if the tag changed, the key would too. I assume this was a find/replace that should not have touched this file. Please revert this file.

Reply:
Yes, it was a  find/replace 😅